src/webapi/request_animation_frame_handle.rs

```diff
@@ -0,0 +1,23 @@
+/// Struct representing an animation frame request
+/// Use webapi::window::request_animation_frame to create this
+#[derive(Debug)]
+pub struct RequestAnimationFrameHandle {
+    id: u32,
+}
+
+impl RequestAnimationFrameHandle {
+    /// This id comes from here:
+    /// [(Javascript docs)](https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame)
+    pub fn new(id: u32) -> RequestAnimationFrameHandle {
+        RequestAnimationFrameHandle {
+            id,
+        }
+    }
+    /// This cancels an animation frame request
```

Comment:
`This cancels an animation frame request` -> `Cancels an animation frame request.` (:

Also please put an extra newline before `[(Javascript docs)]`